packages/cli/src/ui/hooks/useReactToolScheduler.ts

```diff
@@ -63,7 +63,7 @@ export type TrackedToolCall =
   | TrackedCancelledToolCall;
 
 export function useReactToolScheduler(
-  onComplete: (tools: CompletedToolCall[]) => void,
```

Comment:
wow this is a big fix! A little annoyed no lints triggered when our implementations of this were async methods that would need to be awaited..